orte/tools/orterun/orterun.1in

```diff
@@ -13,10 +13,12 @@
 .SH NAME
 .
 orterun, mpirun, mpiexec \- Execute serial and parallel jobs in Open MPI.
+oshrun, shmemrun \- Execute serial and parallel jobs in Open SHMEM.
```

Comment:
It would be fine to add other synonyms -- but that's not what this line does.  This line completely replaced everything with just the OSHMEM names.